src/index.js

```diff
@@ -5,19 +5,32 @@ function startsWith (key, val) {
   return key.indexOf(val) === 0;
 }
 
+function isAttribute (key) {
+  return key === 'attributes';
+}
+
 function shouldBeAttr (key, val) {
-  return startsWith(key, 'aria-') || startsWith(key, 'data-');
+  return startsWith(key, 'aria-') || startsWith(key, 'data-') || isAttribute(key);
 }
 
 function handleFunction (Fn) {
   return Fn.prototype instanceof HTMLElement ? new Fn() : Fn();
 }
 
+function setAttr (node, attrName, attrValue) {
```

Comment:
I think it might be good to just make this a breaking change and require `attributes` to set attributes (or `attrs`?). Everything else just sets props. Maybe at the same time we can do `events`? I'd like to split this out into a separate library for declaratively constructing real DOM eventually, also.